plugins/org.eclipse.sirius.table/src/org/eclipse/sirius/table/business/internal/refresh/DTableElementSynchronizer.java

```diff
@@ -601,8 +601,6 @@ public void refreshStyle(final DCell cell) {
      * @param style
```

Comment:
For comment of the commit,  explanations are welcome. In what do you simplify this class? The "simplification" only concerns "refactoring" or is there something change on the logic ?

The comment title could be a little bit more explicit: "[cleanup] Simplify style update of DTableElementSynchronizer"

Reply:
It is mistake: this commit should belong to ticket #116.